### ISSUE TITLE
Content nodes generated by `dotnet migrate` aren't read by `dotnet publish`

### ISSUE THREAD

**Issue by BillWagner** [dotnet/cli#5753]
## Steps to reproduce

1. Run `dotnet migrate` on a project where the `project.json` file includes a publishOptions node that has additional directories or files.:

```json
"publishOptions": {
  "include": [
    "wwwroot",
    "Pics",
    "Dockerfile"
}
```
Run `dotnet migrate` on this project.

## Expected  behavior

The `ItemGroup` node in the CSProj should include separate `Content` elements for each item in the `publishOptions` group:

```xml
<ItemGroup>
  <Content Include="wwwroot\**\*;">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
  </Content>
  <Content Include="Pics\**\*;">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
  </Content>
  <Content Include="Dockerfile;">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
  </Content>
</ItemGroup>
```

## Actual behavior

The `csproj` contains one `Content` node with all paths in it:

```xml
<ItemGroup>
  <Content Update="wwwroot\**\*;Pics\**\*;Dockerfile">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
  </Content>
</ItemGroup>
```

`dotnet publish` does not publish the nodes correctly.

## Environment data
`dotnet --info` output:

> .NET Command Line Tools (1.0.0-rc4-004812)
> 
> Product Information:
>  Version:            1.0.0-rc4-004812
>  Commit SHA-1 hash:  9cf8e8c57f
> 
> Runtime Environment:
>  OS Name:     Mac OS X
>  OS Version:  10.12
>  OS Platform: Darwin
>  RID:         osx.10.12-x64
>  Base Path:   /usr/local/share/dotnet/sdk/1.0.0-rc4-004812

/cc @blackdwarf

**Comment by blackdwarf** [comment]
The issue here is that the web SDK implies globs in addition to the base SDK. The actual right fix for this would be to create `<Content>` nodes with `Update` syntax only for those that we know are implied in the web sdk and `Include` for the rest of them.

/cc @livarcocc @mlorbetske

**Comment by livarcocc** [comment]
I don't want migration to be aware of the blobs in the SDKs, that will lead to only pain when maintaining this code. When we made this change to update instead of include, it was my understanding that the Web.SDK was pretty much doing a include all.

@mlorbetske did this change?

**Comment by mlorbetske** [comment]
@livarcocc the only change is that core SDK has the include everything behavior rather than Web SDK & Web SDK only updates on core SDK's "include everything" now. There shouldn't be a change in behavior as far as migration is concerned, this happened for RC4

**Comment by livarcocc** [comment]
@mlorbetske So, should what migrate is doing above work?

**Comment by mlorbetske** [comment]
I believe so, unless `Pics\**\*` and `Dockerfile` aren't item type `Content`

/cc @vijayrkn @srivatsn

**Comment by blackdwarf** [comment]
@mlorbetske I could be missing something, but if there is no `<Content Include="Pics\**\*" />` anywhere in the project (including any imports), how would `Update` work? It is not an upsert, meaning, it will not add a metadata item if it doesn't exist, right?

**Comment by mlorbetske** [comment]
@blackdwarf this should be covered by a `**\*` glob in the core SDK, the open question is whether that glob has an item type of `Content` or if it's `None`

**Comment by livarcocc** [comment]
I chatted with @dsplaisted offline and we reached a conclusion here.

@mlorbetske the NET SDK is using None for their globbing. That's why the Content Update is not working above.

What we talked about doing is making the CLI not migrate items that are Content Include in the Web SDK (I really wanted to avoid this, but I will see if this doesn't look to horrible) and for everything else to do a None Update.

Any concerns with this approach?
